Azure/communication-ui-library-android
# [Feature] support device name, and hide receiver when bluetooth active

## Purpose

* Hides "Receiver/Android" when Bluetooth is connected, and shows the Bluetooth device name

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
Connect a bluetooth device, verify "Receiver/Android" disapears, Verify correct device name is shown.
```

## What to Check
Verify that the following are valid
* ...

## Other Information
